chains/metal.json

```diff
@@ -0,0 +1,23 @@
+{
+  "alias": "metal",
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "required": false
+      },
+      "url": "https://explorer.metall2.com/api"
+    },
+    "browserUrl": "https://explorer.metall2.com/"
```

Comment:
Note to myself: It's op-stack, that's why [GasPriceOracle](https://explorer.metall2.com/address/0x420000000000000000000000000000000000000F) exists. However it's better to check whether this contract is being actively utilized.